### PR TITLE
Align expiration,cvv in row and handle capture error

### PR DESCRIPTION
**Pull request by prem548**
What?
   1) we should handle all Errors from Order calls in a similar fashion - no special exception case is needed for `INSTRUMENT_DECLINED`
   2) Align `Expiration Date and `CVV` in a row. 
   3) Use `const` instead of `var`
   
Why?
      Existing "INSTRUMENT_DECLINED" has `return actions.restart();` when Orders calls return the declined error- in that case we would see `undefined actions parameter` error. After a lot of discussions get to know that we got confirmation to handle all errors using the existing "errorDetail" condition.

### CODE REVIEW

File: advanced-integration/public/app.js [hunk 1]
```diff
@@ -115,18 +115,13 @@ if (paypal.HostedFields.isEligible()) {
...
+              // Two cases to handle:
+              //   (1) Other non-recoverable errors -> Show a failure message
+              //   (2) Successful transaction -> Show confirmation or thank you
               // This example reads a v2/checkout/orders capture response, propagated from the server
               // You could use a different API or structure for your 'orderData'
-              var errorDetail =
+              const errorDetail =
                 Array.isArray(orderData.details) && orderData.details[0];
-              if (errorDetail && errorDetail.issue === "INSTRUMENT_DECLINED") {
-                return actions.restart(); // Recoverable state, per:
```
Comment (gregjopa) [inline advanced-integration/public/app.js:127]: Great catch. This `actions.restart()` feature works for the PayPal button to restart the flow so the buyer can try again with a different funding instrument. It does not exist in hosted fields.

**Review by gregjopa — APPROVED** [review]

**Comment by xjamundx** [comment]
Can you throw in a screenshot of the new style. In general I'd rather not use inline styles, but since there's no stylesheets in this repo that works fine :D

File: advanced-integration/views/checkout.ejs [hunk 2]
```diff
@@ -18,13 +18,15 @@
...
         <label for="card-number">Card Number</label>
         <div id="card-number" class="card_field"></div>
-        <div>
-          <label for="expiration-date">Expiration Date</label>
-          <div id="expiration-date" class="card_field"></div>
-        </div>
-        <div>
-          <label for="cvv">CVV</label>
-          <div id="cvv" class="card_field"></div>
+        <div style="display: flex; flex-direction:row;">
```
Comment (xjamundx) [inline advanced-integration/views/checkout.ejs:21]: 1 space after `:` please in each of these for consistency

File: advanced-integration/views/checkout.ejs [hunk 3]
```diff
@@ -18,13 +18,15 @@
...
+          <div>
+            <label for="expiration-date">Expiration Date</label>
+            <div id="expiration-date" class="card_field"></div>
+          </div>
+          <div style="margin-left:10px;">
```
Comment (xjamundx) [inline advanced-integration/views/checkout.ejs:26]: same here `margin-left: 10px`

**Comment by prem548** [comment]
Error case: 
Screenshot 2022-07-01 at 1 32 44 AM

Orders call response: 
Screenshot 2022-07-01 at 1 33 09 AM

VPM response
Screenshot 2022-07-01 at 1 33 24 AM

Capture response 
Screenshot 2022-07-01 at 1 33 34 AM

**Comment by prem548** [comment]
rendering CVV in row
Screenshot 2022-07-01 at 1 38 08 AM

**Comment by prem548** [comment]
Success Case: 
Screenshot 2022-07-01 at 1 40 21 AM

File: advanced-integration/public/app.js (hunk 1 above)
Comment (shango420) [reply advanced-integration/public/app.js:127]: advanced-integration/views/checkout.ejs

**Review by shango420 — APPROVED** [review]

**Review by shango420 — COMMENTED** [review]
a980ca20acccf9854b3bbb4c2f3e6321e3003bf5

**Pull request merged**